open-sphere-base/mantle/src/main/java/io/opensphere/mantle/icon/impl/gui/IconChooserPanel.java

```diff
@@ -149,6 +149,12 @@
     /** The optional selected icon URL. */
     private String mySelectedUrl;
 
+    /** The icon display width */
+    private int myTileWidth;
+
+    /** The list of icons being resized */
+    private List<IconRecord> myResizeRecord = New.list();

```

Comment:
Add 's' or 'List' to the end to indicate it's a collection

Reply:
resolved

Reply:
It's still called 'myResizeRecord'.  Also the get/set methods should be renamed in the same way